jf_agent/jf_jira/jira_download.py

```diff
@@ -31,16 +31,15 @@ def download_users(
     jira_connection, gdpr_active, quiet=False, required_email_domains=None, is_email_required=False
 ):
     if not quiet:
-        print('downloading jira users... ', end='', flush=True)
```

Comment:
Q: Do we need this functionality? It looks like you didn't include above (instead just printing out a new log line), but do have this here.

Reply:
It's nice to compact the output, from a UX sense. It _doesn't_ work if the process is big and warrants a TQDM progress bar... I ended up going back and doing the check mark appraoch for all git processes too (where applicable). The new output will look like this (in stdout). Finishing up some changes and then I'll push the latest changes

```
Obtained Jira configuration, attempting download...
downloading jira fields... ✓
downloading jira projects... ✓
downloading jira project components... ✓
downloading jira versions... ✓
downloading jira users...  ✓
downloading jira resolutions... ✓
downloading jira issue types...  ✓
downloading jira issue link types... ✓
downloading jira priorities... ✓
downloading jira boards...: 100%|██████████| 3/3 [00:00<00:00,  3.44it/s]
downloading jira sprints: 100%|██████████| 4/4 [00:04<00:00,  1.07s/it]
downloading issue metadata... ✓
downloading jira issues: 100%|██████████| 12633/12633 [02:54<00:00, 72.21it/s]
downloading jira worklogs...  ✓
downloading jira custom field options... ✓
downloading jira statuses... ✓

Starting Git download for 1 provided git configurations
downloading github users... ✓
downloading github projects... ✓
downloading github repos... ✓
downloading commits on branch develop for jellyfish: 524commits [00:06, 84.24commits/s]
downloading commits on branch main for jf_ingest: 26commits [00:01, 19.35commits/s]
downloading PRs for jellyfish: 18prs [00:37,  2.08s/prs]
downloading PRs for jf_ingest: 1prs [00:02,  2.66s/prs]

```